docs/changes/newsfragments/5932.improved_driver

```diff
@@ -0,0 +1,5 @@
+Fix spelling of the 2 incorrectly spelled classes
+```qcodes.instrument_drivers.tektronix.TekronixDPOTrigger``` and
+ ```qcodes.instrument_drivers.tektronix.TekronixDPOWaveform``` to
+ ```qcodes.instrument_drivers.tektronix.TektronixDPOTRigger```
```

Comment:
```suggestion
 ```qcodes.instrument_drivers.tektronix.TektronixDPOTrigger```
```

Reply:
So much for copilot helping with writing docs ;)